test/e2e/adminUI/adminUI.js

```diff
@@ -151,14 +151,14 @@ module.exports = {
 							value: '.EditForm-container .field-type-text[for="name"] input[name="name"]',
 						},
 						fieldA: {
-							button: '.EditForm-container .field-type-boolean:nth-child(5) label button',
-							label: '.EditForm-container .field-type-boolean:nth-child(5) label span',
-							value: '.EditForm-container .field-type-boolean:nth-child(5) label input[name="fieldA"]'
+							button: '.EditForm-container .field-type-boolean[for="fieldA"] button',
+							label: '.EditForm-container .field-type-boolean[for="fieldA"] span',
+							value: '.EditForm-container .field-type-boolean[for="fieldA"] input[name="fieldA"]'
 						},
 						fieldB: {
-							button: '.EditForm-container .field-type-boolean:nth-child(6) label button',
-							label: '.EditForm-container .field-type-boolean:nth-child(6) label span',
-							value: '.EditForm-container .field-type-boolean:nth-child(6) label input[name="fieldB"]'
+							button: '.EditForm-container .field-type-boolean[for="fieldB"] button',
+							label: '.EditForm-container .field-type-boolean[for="fieldB"] span',
+							value: '.EditForm-container .field-type-boolean[for="fieldB"] input[name="fieldB"]'
 						}
 					}
```

Comment:
minor comment here...@JedWatson want tests to eventually pass the same linting rules applied to keystone core, one of which is comma dangling.  Please add a trailing comma to to all JSON lines...see others for example...tnx